packages/@aws-cdk/aws-config/README.md

```diff
@@ -1,2 +1,72 @@
 ## The CDK Construct Library for AWS Config
 This module is part of the [AWS Cloud Development Kit](https://github.com/awslabs/aws-cdk) project.
+
+Supported:
+* Config rules
+
+Not supported
+* Configuration recoder
+* Delivery channel
+* Aggregation
+
+### Rules
+
+#### AWS managed rules
+To set up a managed rule, define a `ManagedRule` and specify its identifier:
+
+```ts
+new ManagedRule(this, 'AccessKeysRotated', {
+  identifier: 'ACCESS_KEYS_ROTATED'
+});
+```
+
+Available identifiers and parameters are listed in the [List of AWS Config Managed Rules](https://docs.aws.amazon.com/config/latest/developerguide/managed-rules-by-aws-config.html).
+
+
+Higher level constructs for managed rules are available, see [Managed Rules](lib/managed-rules.ts). Prefer to use those constructs when available (PRs welcome to add more of those).
```

Comment:
This link will only work on GitHub, not on the docs site. Prefer putting in the full GitHub `https://` URL.